dbms/src/Common/ThreadStatus.h

```diff
@@ -62,6 +64,8 @@ class ThreadGroupStatus
     UInt32 master_thread_number = 0;
     Int32 master_thread_os_id = -1;
 
+    std::atomic<LogsLevel> client_logs_level = LogsLevel::none;
```

Comment:
No need for atomic?

Reply:
I thought that we need atomic here because client_logs_level could be set in one thread while the other thread wants to produce logs (the first thread will write and the second will read).